weave/trace_server/migrations/006_seed_costs.up.sql

```diff
@@ -0,0 +1,506 @@
+INSERT INTO llm_token_prices 
+    (id, pricing_level, pricing_level_id, provider_id, llm_id, effective_date, prompt_token_cost, prompt_token_cost_unit, completion_token_cost, completion_token_cost_unit, created_by, created_at) 
+VALUES 
+    (generateUUIDv4(), 'default', 'default', 'openai', 'gpt-4', now(), 3e-05, 'USD', 6e-05, 'USD', 'system', now()), 
```

Comment:
please use UUIDv7 https://clickhouse.com/docs/en/sql-reference/functions/uuid-functions#generateUUIDv7

Reply:
i believe our version of clickhouse is too old for v7

Reply:
whaaa? according to what?